README.md

```diff
@@ -1 +1,21 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+Project Name
+ setting up my Hello Microverse project
+
+
+👤 Author
+
+GitHub: @basitali111
+Twitter: @BasitAl35031734
+LinkedIn: BasitAl35031734
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Please use [this template](https://github.com/microverseinc/readme-template) to make you README.md file descriptive, but remember to customize it to your project.